server/src/main/java/com/hedera/block/server/producer/ProducerBlockItemObserver.java

```diff
@@ -157,42 +160,47 @@ public ProducerBlockItemObserver(
     public void onNext(
             @NonNull final com.hedera.hapi.block.protoc.PublishStreamRequest publishStreamRequest) {
 
-        try {
-            LOGGER.log(DEBUG, "Received PublishStreamRequest from producer");
-            final BlockItem blockItem =
-                    toPbj(BlockItem.PROTOBUF, publishStreamRequest.getBlockItem().toByteArray());
-            LOGGER.log(DEBUG, "Received block item: " + blockItem);
+        LOGGER.log(DEBUG, "Received PublishStreamRequest from producer");
+        final List<BlockItem> blockItemsPbj = new ArrayList<>();
+        final Counter liveBlockItemsReceived = metricsService.get(LiveBlockItemsReceived);
+        for (final var blockItemProtoc : publishStreamRequest.getBlockItemsList()) {
```

Comment:
Personally I would avoid ever using `var`, but not a blocker

Reply:
I generally agree, but in rare instances it fixes the thoroughly excessive wrapping caused by the current spotless configuration.
When it saves multiple line wraps, and is not ambiguous, I would argue that it is acceptable.

Reply:
✅ Done!